core/src/main/resources/jenkins/model/Jenkins/_scriptText.jelly

```diff
@@ -26,8 +26,8 @@ THE SOFTWARE.
   Called from doScriptText() to display the execution result.
 -->
 <?jelly escape-by-default='true'?>
-<st:compress xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
```

Comment:
Compression removed here for a couple reasons:

1. There are like 3 different areas (possibly more) where gzip is enabled.
2. Using compression on user-supplied input can be abused to leak information about HTTPS connections, though this is not that likely here since an admin would already have full control over Jenkins.